src/prg_modelham_mod.F90

```diff
@@ -118,8 +118,8 @@ end subroutine prg_parse_mham
   subroutine prg_twolevel_model(ea, eb, dab, daiaj, dbibj, dec, rcoeff, reshuffle, &
    & seed, h_bml, verbose)
     real(dp), intent(in) :: ea, eb, dab, daiaj, dbibj, rcoeff
-    integer, intent(in) :: verbose
-    integer :: seed
+    integer, intent(in) :: verbose, seed
+    integer :: seedin(33)
```

Comment:
Maybe my Fortran skills are getting rusty :smile: I don't understand why you want an integer array here. The `seed` argument is a scalar.